guides/advanced_guides/search_parameters.md

```diff
@@ -18,11 +18,15 @@ Search parameters let the user customize their search request.
 
 ## Query (q)
 
-The query parameter is the **only mandatory** parameter. This is the string used by the search engine to find relevant documents.
+This is the string used by the search engine to find relevant documents.
+
+When q isn't specified, a placeholder search is performed, returning documents that best match other search parameters.
+A placeholder allows you to find documents without providing keywords. It will return documents sorted according to the relevant [ranking rules](/guides/advanced_guides/settings.md#ranking-rules).
```

Comment:
```suggestion
A placeholder allows you to find documents without providing keywords. It will return documents sorted according to the [ranking rules](/guides/advanced_guides/settings.md#ranking-rules) order.
```

Is keyword the best word to describe the query string?

Reply:
What about 'search terms'?